### PR TITLE
Require SLE12 SP1 or newer for OpenStack roles on SUSE

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: crowbar_framework/app/models/nova_service.rb [hunk 1]
```diff
@@ -33,6 +33,7 @@ def role_constraints
           "unique" => false,
           "count" => 1,
           "exclude_platform" => {
+            "suse" => "< 12.1",
             "windows" => "/.*/"
           },
           "cluster" => true
```
Comment (vuntz) [inline crowbar_framework/app/models/nova_service.rb]: Can you update it for docker also (the role below)?

File: crowbar_framework/app/models/nova_service.rb [hunk 2]
```diff
@@ -77,6 +81,7 @@ def role_constraints
           "unique" => false,
           "count" => 1,
           "exclude_platform" => {
+            "suse" => "< 12.1",
             "windows" => "/.*/"
           }
         },
```
Comment (vuntz) [inline crowbar_framework/app/models/nova_service.rb]: Missing exclusion for xen (below)

File: crowbar_framework/app/models/nova_service.rb (hunk 1 above)
Comment (dirkmueller) [inline crowbar_framework/app/models/nova_service.rb]: I wasn't sure yet what the docker plan is.. thats why I left it out.

File: crowbar_framework/app/models/nova_service.rb (hunk 2 above)
Comment (dirkmueller) [inline crowbar_framework/app/models/nova_service.rb]: added

**Comment by vuntz** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**